src/suggest-list.jsx

```diff
@@ -17,6 +17,14 @@ export default class SuggestList extends React.Component {
     return this.props.isHidden || this.props.suggests.length === 0;
   }
 
+  componentWillReceiveProps(newProps) {
+    if (newProps !== this.props) {
```

Comment:
Isn’t a deeper comparison needed for these two objects? This just checks the reference…